test.js

```diff
@@ -706,3 +706,49 @@ test('calling cancel method on a process which has been killed does not make err
 	const {isCanceled} = await t.throwsAsync(subprocess);
 	t.false(isCanceled);
 });
+
+test.cb('fork()', t => {
+	const fork = execa.fork('fixtures/noop');
+
+	fork.on('close', code => {
+		t.is(code, 0);
```

Comment:
We removed the `code` property.

Reply:
`code` was removed from the result object return by `execa`, but listening to event still return the appropriate code.
Anyway, both solutions works, but I prefer yours since it match with what's already done.

Reply:
See #250. We removed it in tests too.